### DISCUSSION TITLE
Problems configuring guscis.json in hugo.

### DISCUSSION THREAD

**Discussion by anghunk** [giscus/giscus#755] [Q&A] [answered]
In hugo, where should the `giscus.json` configuration file be placed, I have tried many times, but it doesn't work.By convention, I put it in the `/content` folder. `public` is generated normally, but in fact the web page does not take effect for the configuration.

**Comment by ouuan** [comment, marked as answer]
`giscus.json` should be placed in the root of the repository whose Discussions is used for giscus. It doesn't need to be hosted on your website, so it's not in `/content` or `/static`.

It's used to control how giscus can interact with your repository. If you think about it, you can know that it doesn't make sense to let your website host it.

**Reply by anghunk** [reply]
> `giscus.json` should be placed in the root of the repository whose Discussions is used for giscus. It doesn't need to be hosted on your website, so it's not in `/content` or `/static`.
> 
> It's used to control how giscus can interact with your repository. If you think about it, you can know that it doesn't make sense to let your website host it.

Yes, I see, I should put this file in the root of the github repository where discussions work, I really appreciate your reply, it has been bothering me for a long time!
